beets/plugins.py

```diff
@@ -418,10 +418,14 @@ def queries() -> dict[str, type[Query]]:
     return out
 
 
-def types(model_cls: type[T]) -> dict[str, type[Any]]:
+if TYPE_CHECKING:
```

Comment:
We've got this in three places now. Can we unify all type definitions at the top of the module instead?

Reply:
Up to you. I like having them close to the place where types are used but I see that placing them at top would dedupe a bit.

Reply:
I think I'd prefer having just a single place where they are defined.

Note, I invited you to join our organisation Zulip chat!